### PR TITLE
Added getBrokerUsers call before silent to get uid based on login_hint

### PR DESCRIPTION
**Pull request by kreedula**
To get this test working : https://identitydivision.visualstudio.com/Engineering/_workitems/edit/926501

### CODE REVIEW

File: userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java [hunk 1]
```diff
@@ -170,9 +177,32 @@ public void onAcquireTokenSilentClicked(final AcquireTokenFragment.RequestOption
...
+                        final UserInfo[] userInfos = mAuthContext.getBrokerUsers();
+
+                        for (UserInfo userInfo : userInfos) {
+                            if (userInfo.getDisplayableId().equalsIgnoreCase(requestOptions.getLoginHint())) {
+                                uId = userInfo.getUserId();
+                                break;
+                            }
+                        }
+                    } catch (OperationCanceledException | AuthenticatorException | IOException e) {
+                        Logger.e(TAG, "Unable to getbrokerusers from broker " + e.getMessage());
```
Comment (iambmelt) [inline userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java]: Can we `Toast` this message?

If this fails and its not hooked up to a debugger, it'll just look like nothing happened

Comment (iambmelt) [reply userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java]: Do we still want to try the silent call if `getBrokerUsers` fails like this? Seems like we should check for a non-error result before proceeding

Comment (kreedula) [reply userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java]: Only proceeding with silent if uId is non-null else showing a toast

**Comment by shahzaibj** [comment]
Did this test case ever used to work? If yes, then how?

**Comment by kreedula** [comment]
> Did this test case ever used to work? If yes, then how?

Looking at the code, i don't think it could have ever worked

**Pull request merged**